pkg/daemon/update.go

```diff
@@ -512,6 +512,16 @@ func (dn *Daemon) update(oldConfig, newConfig *mcfgv1.MachineConfig) (retErr err
 	return dn.finalizeAndReboot(newConfig)
 }
 
+/// removeRollback removes the rpm-ostree rollback deployment.  It
+/// takes up space, and we don't generally expect administrators to
+/// use this versus e.g. removing broken configuration.  We only
+/// remove the rollback once the MCD pod has landed on a node, so
+/// we know kubelet is working.
+func (dn *Daemon) removeRollback() error {
+	_, err := runGetOut("rpm-ostree", "cleanup", "-r")
```

Comment:
haven't checked myself, does rpm-ostree return success if there is no rollback deployment available to delete?

Reply:
Yep, it's a no-op if not present.